db/re/item_db_etc.yml

```diff
@@ -38895,7 +38895,7 @@ Body:
     Script: |
       .@atk = 5;
       .@r = getrefine();
-      if (getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW) == W_SPEAR && .@r >= 10) {
+      if ((getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW) == W_1HSPEAR || getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW) == W_2HSPEAR) && .@r >= 10) {
```

Comment:
Please store the value retrieved by getiteminfo in a variable like with refine if you need it more than once.

Reply:
Thanks!